space-template.yaml

```diff
@@ -0,0 +1,166 @@
+---
+# These define the potential field types that ca be used in the fields of work
+# item types.
+field_types:
+  - &string string
+  - &translatable_string translatable_string
+  - &list list
+  - &timestamp timestamp
+  - &user user
+
+# A map of common strings that can be used to denote states of a work item
+common_work_item_states: &common_work_item_states
+  &state_new new:
+      en_US: "new"
+      de_DE: "neu"
+  &state_open open:
+      en_US: "open"
+      de_DE: "offen"
+  &state_closed closed:
```

Comment:
how does the system know which states are the actual closed one ? since i.e. when filtering backlog lists you dont want to see the issues in a state that resembles closed/resolved, etc.